src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -245,6 +245,26 @@ internal static partial class Library
                     returnBehavior: ReturnBehavior.ReturnBlankIfAnyArgIsBlank,
                     targetFunction: CountRows)
             },
+            {
+                BuiltinFunctionsCore.CountA,
+                StandardErrorHandling<TableValue>(
+                    expandArguments: NoArgExpansion,
+                    replaceBlankValues: DoNotReplaceBlank,
+                    checkRuntimeTypes: ExactValueTypeOrBlank<TableValue>,
+                    checkRuntimeValues: DeferRuntimeValueChecking,
+                    returnBehavior: ReturnBehavior.ReturnBlankIfAnyArgIsBlank,
+                    targetFunction: CountA)
+            },
+            {
+                BuiltinFunctionsCore.Count,
+                StandardErrorHandling<TableValue>(
+                    expandArguments: NoArgExpansion,
+                    replaceBlankValues: DoNotReplaceBlank,
+                    checkRuntimeTypes: ExactValueTypeOrBlank<TableValue>,
+                    checkRuntimeValues: DeferRuntimeValueChecking,
+                    returnBehavior: ReturnBehavior.ReturnBlankIfAnyArgIsBlank,
+                    targetFunction: Count)
+            },
             {
                 BuiltinFunctionsCore.CountRows_UO,
```

Comment:
[minor] Can you please move this up before CountIf, to keep the functions sorted?